src/components/action-popover/action-popover.stories.mdx

```diff
@@ -82,12 +84,170 @@ import {
 
 ## Examples
 
+## TEST_TEST
```

Comment:
Don't be alarmed by this story. It's here to locally replicate the code used in the Github Issue. Makes manually testing the bug easier.

Reply:
I've moved this story and renamed it as there's some value in having it IMO, I'll catch up with QA about whether cypress tests can be added